core/templates/dev/head/pages/about-page/about-page.mainpage.html

```diff
@@ -1,636 +1,630 @@
-{% extends 'dist/base.html' %}
-
-{% block maintitle %}
-  About us - Oppia
-{% endblock maintitle %}
-
-{% block header_js %}
-  {{ super() }}
-{% endblock %}
-
-{% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      About
-    </li>
-  </ul>
-{% endblock navbar_breadcrumb %}
-
-{% block content %}
-  <div class="oppia-static-header">
-    <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
-  </div>
-  <background-banner></background-banner>
-  <div ng-controller="About">
-    <div class="oppia-static-content-below-banner">
-      <div class="oppia-page-card oppia-static-content">
-        <div class="about-nav md-tab">
-          <ul class="oppia-about-tabs">
-            <li class="oppia-about-tabs-active">
-              <a ng-click="onTabClick(TAB_ID_ABOUT)" href="" class="oppia-about-tabs-text" id="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
-            </li>
-            <li>
-              <a ng-click="onTabClick(TAB_ID_FOUNDATION)" href="" class="oppia-about-tabs-text" id="foundation" translate="I18N_ABOUT_PAGE_TABS_FOUNDATION"></a>
-            </li>
-            <li>
-              <a ng-click="onTabClick(TAB_ID_CREDITS)" href="" class="oppia-about-tabs-text" id="credits" translate="I18N_ABOUT_PAGE_TABS_CREDITS"></a>
-            </li>
-          </ul>
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    @require('../../base_components/header.html', {"title": "About us - Oppia"})
```

Comment:
Is it possible to use absolute paths, or paths relative to a global base path?

Reply:
It isn't currently supported by the webpack loader. However, I've created an issue where I will be working on this.

Reply:
OK. Can you link to the issue? Always do that when responding to comment threads relating to filing issues etc. so that they can be considered resolved.

Reply:
Okay, here is the link to the issue #6847.